external/common/murmurhash3/MurmurHash3.cpp

```diff
@@ -10,15 +10,15 @@ namespace nupic {
  * Don't worry about the technically undefined behavior when r >= 32, since this
  * is only used with a hardcoded r
  */
-inline UInt32 rotl32 ( UInt32 x, char r )
+inline unsigned int rotl32 ( unsigned int x, char r )
```

Comment:
this probably should be explicitely 32bit type, so unsigned int32_t 
https://en.cppreference.com/w/cpp/types/integer

Reply:
> this probably should be explicitely 32bit type

yes, good catch.